content/docs/0.13.x/operate/user_management/oauth_secrets/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: OpenID Session Secrets
+description: Manage session and database encryption secret of your Keptn Bridge.
+weight: 30
+---
+
+In this section, the management of the session and database encryption secrets of the Keptn Bridge is explained.
+
+- `session secret`: This secret is used to hash the session of the user.
+- `database encryption secret`: This secret is used to encrypt the session in the database.
+
+## Create session and database encryption secret
+
+* To create the session and database encryption secret, use the following commands provided by the operating system:
+
+    <details><summary>on Linux </summary>
+    <p>
+
+    * To generate the secrets, use the following commands and store them into the environment variables `OAUTH_SESSION_SECRET` and `OAUTH_DATABASE_ENCRYPT_SECRET`:
```

Comment:
```suggestion
    * Use the following commands to generate the secrets and store the results into the environment variables `OAUTH_SESSION_SECRET` and `OAUTH_DATABASE_ENCRYPT_SECRET`:
```
The existing syntax says that one is storing the commands in the environment variables and I don't think that is the intent.